.env.example

```diff
@@ -49,8 +49,8 @@ NORTHSTAR_AUTH_SECRET=
 NORTHSTAR_CLIENT_ID=
 NORTHSTAR_CLIENT_SECRET=
 
-PHOENIX_URI=https://staging.dosomething.org
```

Comment:
We still have a `PHOENIX_URI` in here right? except now it is set to next?

Reply:
oh yes I forgot to update the `.env.example`! I'll do that now!